contracts/FlowContractAudits.cdc

```diff
@@ -102,35 +102,35 @@ pub contract FlowContractAudits {
         }
 
         // Remove a voucher with given key
-        pub fun deleteVoucher(key: String) {
+        access(all) fun deleteVoucher(key: String) {
             FlowContractAudits.deleteVoucher(key)            
         }
     }
 
     // Used by admin to set the Auditor capability
-    pub resource interface AuditorProxyPublic {
-        pub fun setAuditorCapability(_ cap: Capability<&Auditor>)
+    access(all) resource interface AuditorProxyPublic {
+        access(all) fun setAuditorCapability(_ cap: Capability<&Auditor>)
     }
 
     // The auditor account will have audit access through AuditorProxy
     // This enables the admin account to revoke access
     // See https://docs.onflow.org/cadence/design-patterns/#capability-revocation
-    pub resource AuditorProxy: AuditorProxyPublic {
+    access(all) resource AuditorProxy: AuditorProxyPublic {
         access(self) var auditorCapability: Capability<&Auditor>?
 
-        pub fun setAuditorCapability(_ cap: Capability<&Auditor>) {
+        access(all) fun setAuditorCapability(_ cap: Capability<&Auditor>) {
             self.auditorCapability = cap
         }
 
-        pub fun addVoucher(address: Address?, recurrent: Bool, expiryOffset: UInt64?, code: String) {
+        access(all) fun addVoucher(address: Address?, recurrent: Bool, expiryOffset: UInt64?, code: String) {
```

Comment:
Judging by the `AuditorProxyPublic` interface, looks like there's an `AuditorProxy` Capability published - should these privileged methods then be entitled?

Reply:
I probably should just remove this contract. I think we can just let it break because it is not used any more